hud/powerdisplay.lua

```diff
@@ -49,6 +49,7 @@ end
 --Only the glass proxy is required, rest have default values.
 function powerDisplay.widget(glasses, data)
     if data ~= nil then
+    if data.state ~= states.MISSING then
```

Comment:
`powerStatus` does not have a `state` property

Reply:
Then there's something wonky because powerStatus does have a state property.

Reply:
Damn

Reply:
True dat, my mistake